front-end/src/components/cookie/CookieView.scss

```diff
@@ -0,0 +1,83 @@
+%common-position-rule {
+    position: fixed;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+}
+
+%common-font-rules {
+    font: {
+        family: GT-Walsheim;
+        style: normal;
+        stretch: normal;
+    }
+    color: #4a4a4a;
+}
+
+.cookie-modal-backdrop {
+    @extend %common-position-rule;
+
+    z-index: 1040;
+    background-color: rgba(74, 74, 74, 0.7);
+}
+
+.cookie-modal {
```

Comment:
There is a modal implementation from the previous project. You might want to reuse it

Reply:
It is here:
https://github.com/City-of-Vantaa-SmartLab/CultureEvents/blob/master/frontend/src/components/modal/index.js

Reply:
Thanks, will check it.

Reply:
As we discussed, we can go with Bootstrap modal as is used here.